GVFS/GVFS/CommandLine/SparseVerb.cs

```diff
@@ -240,23 +240,32 @@ protected override void Execute(GVFSEnlistment enlistment)
 
         private void PruneFoldersOutsideSparse(ITracer tracer, Enlistment enlistment, SparseTable sparseTable)
         {
-            string[] directoriesToDehydrate = this.GetDirectoriesOutsideSparse(enlistment.WorkingDirectoryBackingRoot, sparseTable);
+            string[] directoriesToDehydrate = new string[0];
+            if (!this.ShowStatusWhileRunning(
+                () =>
+                {
+                    directoriesToDehydrate = this.GetDirectoriesOutsideSparse(enlistment.WorkingDirectoryBackingRoot, sparseTable);
+                    return true;
+                },
+                "Finding folders to prune"))
+            {
+                this.ReportErrorAndExit(tracer, "Failed to prune.");
+            }
+
+            this.WriteMessage(tracer, $"Found {directoriesToDehydrate.Length} folders to prune.");
+
             if (directoriesToDehydrate.Length > 0)
             {
-                if (!this.ShowStatusWhileRunning(
-                    () =>
+                ReturnCode verbReturnCode = this.ExecuteGVFSVerb<DehydrateVerb>(

```

Comment:
Should the console messages in Dehydrate either be parameterized or made more general?  

It looks like this message is always printed:

https://github.com/microsoft/VFSForGit/blob/a537d002be8d48675f48e3e97fe1c8c6359133a9/GVFS/GVFS/CommandLine/DehydrateVerb.cs#L166

And there are some other error messages that use the work "dehydrate":

https://github.com/microsoft/VFSForGit/blob/a537d002be8d48675f48e3e97fe1c8c6359133a9/GVFS/GVFS/CommandLine/DehydrateVerb.cs#L209

Should they say prune (or some verb-neutral word)?

Reply:
Yeah that would make it easier for the user to understand and we would need possibly different terms for each because you would probably want
`WARNING: If you abort the dehydrate after this point, the repo may become corrupt`
to be
`WARNING: If you abort the sparse after this point, the repo may become corrupt`
but 
`Cannot dehydrate: must have a clean git status.`
to be
`Cannot prune: must have a clean git status.`